mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,286 @@
+    description: "A managed alloydb cluster."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "AlloyDB": "https://cloud.google.com/alloydb/docs/"
+      api: "https://cloud.google.com/alloydb/docs/reference/rest/v1beta/projects.locations.clusters/create"
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 10
+          update_minutes: 10
+          delete_minutes: 10
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "clusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: |
+          The location where the alloydb cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::Time
+        name: "createTime"
+        description: |
+          Time the Cluster was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: "updateTime"
+        description: |
+          Time the Cluster was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "User-defined labels for the alloydb cluster."
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "annotations"
+        description: "Annotations to allow client tools to store small amount of arbitrary data. This is distinct from labels."
+      - !ruby/object:Api::Type::String
+        name: "network"
+        required: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: "state"
+        output: true
+        description: |
+          The current state of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "selfLink"
+        output: true
+        description: |
+          The selfLink of the alloydb cluster.
```

Comment:
don't need to define property. Automatically included.
```suggestion
```

Reply:
+1